### DISCUSSION TITLE
HyperHDR on Rpi4 with Libreelec 11 - not grabbing screen

### DISCUSSION THREAD

**Discussion by Sulya** [awawa-dev/HyperHDR#979] [Q&A]
As the title says, I have Rpi4 with Libreelec 11 with latest HyperHDR plugin and less connected via esp32
Startup animation plays fine, but when a movie is playing there is nothing happening with the leds.
In the Video Capturing page only USB Capture seems to be available.
Am I missing something? Thanks for the help

**Comment by awawa-dev** [comment]
For this system only capturing with USB grabber is supported currently.

**Reply by Sulya** [reply]
Oh, thanks. Because I used an RPi 2 with Libreelec and Hyperion 8-ish years ago I assumed this setup would work

**Reply by awawa-dev** [reply]
It's mentioned in the wiki:
![obraz](https://github.com/user-attachments/assets/89b47d28-71f9-4a3f-b1c9-45f00f2e3df0)

Unfortunately Linux has taken such an unfortunate direction of development that there is currently no universal API that would allow us to capture screens across Linux platforms, although HyperHDR supports Wayland Grabber. This is quite the opposite situation to Windows, where DirectX Software Grabber works great (with the obvious exception of HDCP applications such as Nteflix), so a cheap miniPC N100 with Windows 11 could currently be a very interesting alternative to the RPi as a media center platform. And the Intel N100 is a much more modern design than the RPi chipsets.
